server/server.js

```diff
@@ -148,6 +148,15 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.post("/admin/tweets/approvedTweetsOnlyStatus", function(req, res) {
+        try {
```

Comment:
This part doesn't really need a try/catch block since setApprovalMode can't throw anything, but I'll leave it your discretion whether to change it or not before merging :+1: